task_list.gemspec

```diff
@@ -17,6 +17,7 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
 
   gem.add_dependency "rack", "~> 1.0"
+  gem.add_dependency "nokogiri", "~> 1.6.0"
```

Comment:
The tests were failing, because Nokogiri 1.7 no longer supports Ruby 2 and below.

https://travis-ci.org/deckar01/task_list/jobs/196773153

It appears that others have dropped support for Ruby 1.9.3 and 2.0 when they run into this issue.

https://github.com/twingly/twingly-search-api-ruby/issues/61